resources/sync_gateway_configs/xattrs/old_doc_cc.json

```diff
@@ -9,11 +9,15 @@
             {{ no_conflicts }}
             {{ sg_use_views }}
             {{ num_index_replicas }}
+            {{ username }}
+            {{ password }}
+            {{ cacertpath }}
+            {{ certpath }}
+            {{ keypath }}
             "import_docs": "continuous",
             "enable_shared_bucket_access": true,
             "server":"{{ server_scheme }}://{{ couchbase_server_primary_node }}:{{ server_port }}",
             "bucket":"data-bucket",
-            "password": "password",
```

Comment:
interesting. This json file does not have username before. Do you still need to add {{ username }}?can you check once again?

Reply:
removed username and password args.